### PR TITLE
Hello microverse setup 

### PR DESCRIPTION
**Pull request by Mounir-kh**
linters created

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# .gitignore
+node_modules/
+
```
Comment (koomekevo) [inline .gitignore:3]: @Mounir-kh kindly add test.md file to the .gitignore as a project requirement.

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @Mounir-kh, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:heavy_check_mark: Your project has no linter errors.
:heavy_check_mark: Your README is descriptive. 

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly make your PR descriptive by adding a descriptive title and short summary of what the PR aims to do.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mounir-kh** [comment]
Thank you for your review, I appreciate

**Review by Lucas-Masaba — APPROVED** [review]
Hello @Mounir-kh 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._


- [ ] Good job implementing the README 👍 . However, it is the default template with default values. Please modify your README to include the right project name(Hello Microverse), tools, your personal author information and so on. Please work on this as it is a core requirement

- [ ] You are missing the test.md as described in the project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) as seen here 
image
I will approve this project but please work on these as they are core requirements 👍 😄

**Pull request merged**